app/api/diagnostic/route.ts

```
@@ -265,7 +279,7 @@ export async function POST(req: Request) {
 
         // 3. Create diagnostics_sessions record
         const newSessionExpiresAt = new Date(Date.now() + SESSION_TIMEOUT_MINUTES * 60 * 1000);
-        const newAnonymousId = user ? null : (clientAnonymousSessionId || crypto.randomUUID());
+        const newAnonymousId = user ? null : (effectiveAnonymousSessionId || crypto.randomUUID());
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)